db/migrate/20220429051159_add_user_ref_to_posts.rb

```diff
@@ -1,5 +1,6 @@
 class AddUserRefToPosts < ActiveRecord::Migration[7.0]
   def change
-    add_reference :posts, :user, null: false, foreign_key: true
+    add_reference :posts, :user, null: false, index: true
+    add_foreign_key :posts, :users, column: :author_id, on_delete: :cascade
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/167686089-67463c9c-129e-42bc-b0c3-c3d089da2bc6.png)

- [ ] Kindly correct this migration as is throwing an error. :point_up: 
Check this example :point_down: 

```rb 
class AddUserRefToPosts < ActiveRecord::Migration[7.0]
  def change
    add_reference :posts, :author, null: false, foreign_key: { to_table: 'users' }
  end
end 
```